docs/configuration.md

```diff
@@ -1,5 +1,257 @@
+
+### Base options
+
+|Option|Description|Default|
+|---|---|---|
+|`APP_NAME`|The gallery name|`Lychee`|
+|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
+|`APP_URL`|The URL of your gallery (which resolves to the `public/` folder)|`http://localhost`|
+|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
+|`TIMEZONE`|The timezone of your photos|system timezone of server|
+|`LYCHEE_UPLOADS`|Path to uploads directory|`uploads/` inside `public/`|
+|`LYCHEE_UPLOADS_URL`|URL to uploads directory|`uploads/`|
+|`TRUSTED_PROXIES`|Trusted proxy IP addresses|`null`|
+
+### Database options
+
+Lychee supports MySQL/MariaDB, SQLite, PostgreSQL and Microsoft SQL Server as database backends. It's configuration is managed using the `DB_` variables.
+
+#### `DATABASE_URL`
+
+Some hosting providers give you a single URL containing all the information needed to configure your database. Therefore, Lychee has a `DATABASE_URL` option which only needs the database connection type. For example:
+
+```ini
+DB_CONNECTION=mysql
+DATABASE_URL="mysql://root:password@127.0.0.1/forge?charset=UTF-8"
+```
+
+If this applies to your hosting provider, you can skip the other DB configuration steps.
+
+#### MySQL/MariaDB
+
+The configuration is exactly the same for both systems.
+
+|Option|Value it should have|
+|---|---|
+|`DB_CONNECTION`|`mysql`|
+|`DB_HOST`|Host of the database server (if it's running on the same server use `127.0.0.1`)|
+|`DB_PORT`|Port of the database server (default 3306)|
+|`DB_DATABASE`|The name of the database|
+|`DB_USERNAME`|Username of the database user|
+|`DB_PASSWORD`|Password of the database user|
+|`DB_SOCKET`|UNIX socket to DB server, replaces `DB_HOST` and `DP_PORT`|
+|`MYSQL_ATTR_SSL_CA`|Optional and only used when using the `pdo_mysql` extension, file path to the SSL certificate authority|
+
+#### SQLite
+
+|Option|Value it should have|
+|---|---|
+|`DB_CONNECTION`|`sqlite`|
+|`DB_DATABASE`|Path to the database file (default `database/database.sqlite`)|
+
+#### PostgreSQL
+
+|Option|Value it should have|
+|---|---|
+|`DB_CONNECTION`|`pgsql`|
+|`DB_HOST`|Host of the database server (if it's running on the same server use `127.0.0.1`)|
+|`DB_PORT`|Port of the database server (default 5432)|
+|`DB_DATABASE`|The name of the database|
+|`DB_USERNAME`|Username of the database user|
+|`DB_PASSWORD`|Password of the database user|
+
+#### Microsoft SQL Server
+
+> {note} Microsoft SQL Server is not officially supported. It may work, but we can't help you if you have any issues with it that do not affect other DB systems as well.
+
+|Option|Value it should have|
+|---|---|
+|`DB_CONNECTION`|`sqlsrv`|
+|`DB_HOST`|Host of the database server (if it's running on the same server use `127.0.0.1`)|
+|`DB_PORT`|Port of the database server (default 5432)|
```

Comment:
Are you sure it's the same port as for Postgres? A quick grep through `/etc/services` suggests it could be `1433`?